app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseScreen.kt

```diff
@@ -92,6 +92,8 @@ abstract class BaseScreen<K : ScreenKey, B : ViewBinding, M : Parcelable, E, F,
     disposable.add(events().subscribe { event ->
       _viewModel.dispatchEvent(event!!)
     })
+    
+    _viewModel.models.observe(viewLifecycleOwner, uiRenderer()::render)
```

Comment:
Won't this current way of instantiating the renderer create a new instance for every model update?

Reply:
OMG yes, for some reason I thought this was a reference 😂